File: LeetCode/2200/main.cpp
class Solution {
public:
    int minBitFlips(int start, int goal) {
        
        int res = start ^ goal;
        int ans = 0;

        while(res) {
            int lsb = res & 1;
            if(lsb == 1) {
                ans++;
            }
            // Remove the last bit
            res >>= 1;
            cout << endl;
        }
        
        return ans;

    }
};

File: LeetCode/2200/main.py
class Solution:
    def minBitFlips(self, start: int, goal: int) -> int:
        res = start ^ goal
        ans = 0

        while (res):
            lsb = res & 1
            if (lsb == 1):
                ans += 1
            res >>= 1

        return ans
